### DISCUSSION TITLE
It's possible to creata Arcs, semi circles and circles?.

### DISCUSSION THREAD

**Discussion by ariellarraburu** [NetTopologySuite/NetTopologySuite#480] [Q&A]
Hi, I saw a solution using buffer to create circles. But it is not correct because it adds the same number of degrees in all directions, so an oval remains on the map.

In the image, all the vectors are 5 km, but they only coincide with the circle in latitude. The circle is central pointt with a buffer of 5000/111111.

![image](https://user-images.githubusercontent.com/6170164/106695782-107ecd00-65ba-11eb-9a54-1a95924cdfd5.png)

I need to create circles, semi circles and quarter circles that are symmetrical placed on an osm map. it's possible?

**Comment by FObermaier** [comment]
NetTopologySuite assumes planar geometry. If you want to use geographic coordinates you
1. need to reproject your data to some spatial reference system that suits the area covered,
2. build the buffer and
3. reproject the result back to WGS84.

Sorry.

**Reply by ariellarraburu** [reply]
Do you have an example of how to do that with NetTopologySuite? 

Sorry, I'm new in this library and I can't find much help in documentation. I need to be able to do all the geometries in 4326 projection, and then represent them in GM or OSM.
